docs/device-emulation.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Device Emulation
+---
+
+## Introduction
+
+Using device-tests, fwupd can prevent regressions by updating and downgrading firmware on real
```

Comment:
inconsistent with device-tests and device-emulation.  I think pick one and stick with it